### PR TITLE
Enhance node container (Closes #25)

### PR DESCRIPTION
**Pull request by kevinresol**
Closes #25

### CODE REVIEW

**Comment by back2dos** [comment]
Oops, forgot about that. Feel free to merge at your discretion ;)

**Comment by kevinresol** [comment]
Also this should fix #96, summon @gene-pavlovsky

**Comment by gene-pavlovsky** [comment]
@kevinresol 
I'd love to try it, but there seems to be a conflict. Can you solve it pls?

**Comment by kevinresol** [comment]
@gene-pavlovsky see if it works now

**Comment by gene-pavlovsky** [comment]
Thanks. I'll try it on weekend.

File: src/tink/http/containers/NodeContainer.hx [hunk 1]
```diff
@@ -72,6 +97,44 @@ class NodeContainer implements Container {
           },
           failures: failures,//TODO: these need to be triggered
         }));
-      });      
+      }
+      
+      if(untyped server.listening) // .listening added in v5.7.0, not added to hxnodejs yet
```
Comment (gene-pavlovsky) [inline src/tink/http/containers/NodeContainer.hx]: If you dare to use the `github` version of `hxnodejs`:
https://github.com/HaxeFoundation/hxnodejs/pull/104#event-1723257365

Comment (kevinresol) [reply src/tink/http/containers/NodeContainer.hx]: Great, but I think I will keep it as-is until hxnodejs got a proper release. So not to frustrate tink_http users.

Comment (gene-pavlovsky) [reply src/tink/http/containers/NodeContainer.hx]: Sure, makes sense. I still haven't got time to test my project with this.

**Comment by kevinresol** [comment]
Let's move ahead, and break it fast =P

**Pull request merged**